Cargo.toml

```diff
@@ -12,7 +12,7 @@ description = "Fast WHATWG Compliant URL parser"
 documentation = "https://docs.rs/ada-url"
 readme = "README.md"
 keywords = ["url", "parser", "whatwg", "performance"]
-categories = ["parser-implementations", "web-programming", "encoding"]
+categories = ["parser-implementations", "web-programming", "encoding", "no-std::no-alloc", "no-std"]
```

Comment:
Categories are not keywords.

Valid categories here: https://crates.io/categories

Can you use `cargo publish --dry-run --allow-dirty` to check if your Cargo.toml is valid.

Reply:
Yes, these are all valid. The correct list is available at https://crates.io/category_slugs